src/middleware/fixImageUrlsMiddleware.js

```diff
@@ -1,14 +1,25 @@
 
-const urlSearchAlpha = 'https://api-alpha.human-connection.org'
-const urlSearchLocal = 'http://localhost:3000'
+const legacyUrls = [
+  'https://api-alpha.human-connection.org',
+  'https://staging-api.human-connection.org',
+  'http://localhost:3000'
+]
 
 export const fixUrl = (url) => {
-  url = url.replace(urlSearchAlpha, '')
-  url = url.replace(urlSearchLocal, '')
+  legacyUrls.forEach((legacyUrl) => {
+    url = url.replace(legacyUrl, '')
```

Comment:
@roschaefer I think in that case, when you take the uploaded images into account, the replaced url should be `/api`. But that would break the badges at the moment. Maybe we should open a seperate ticket for that?

Reply:
Okay you know what, forget that, the relative url is fine as it matches the backend. We have to add the `/api` part on the frontend side, not the backend.